django/db/utils.py

```diff
@@ -102,6 +104,17 @@ def inner(*args, **kwargs):
         return inner
 
 
+def digest(*args):
```

Comment:
Named it [_hash_generator](https://github.com/django/django/pull/6726/files#diff-803276386305dab54c20b97d3d34b82fR40).